components/constants/webui_url_constants.h

```diff
@@ -42,5 +42,6 @@ extern const char kShieldsPanelHost[];
 extern const char kSidebarBookmarksHost[];
 extern const char kFederatedInternalsURL[];
 extern const char kFederatedInternalsHost[];
+extern const char kContentFiltersPath[];
```

Comment:
as per the discussion here: https://bravesoftware.slack.com/archives/C7VLGSR55/p1654798098187649 - It makes sense for the settings related webui to be inside `/brave/common/` this is what Chromium does as well. `chrome/common` but this cannot be solved in this PR. 

We will eventually move webui constants back to `/brave/common/` because it's related to browser code. I've asked @petemill and it's ok to include this for now.